### DISCUSSION TITLE
A few small questions

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1708] [Q&A] [answered]
1) How do I enable the display of EOF characters.
I know that, this plugin provides interesting me feature https://github.com/lukas-reineke/indent-blankline.nvim, but I don't understand how I can enable this according to Nvchad config.

2) Why plugins Gitsigns don't work ? Its commands also do nothing. Maybe I should somehow enable it?

3) I installed https://github.com/simrat39/symbols-outline.nvim and cursor in buffer (see below) to have inconspicuous view. 
Maybe do you know how to change its highlighting?

![image](https://user-images.githubusercontent.com/105173528/206759342-286e85c8-7768-422f-995f-45d943e3e951.png)

UPD : Gitsigns work but they overlapped by diagnostic icons. Perhaps I can display git signs closer than icons.

**Comment by siduck** [comment, marked as answer]
@KirovAI `vim.opt.fillchars = ""` in your custom init.lua file
